apps/web/src/pages/Applications/ApplicationQuestionsPage/ApplicationQuestionsPage.tsx

```diff
@@ -12,7 +17,12 @@ import {
 import useRoutes from "~/hooks/useRoutes";
 import { GetApplicationPageInfo } from "~/types/poolCandidate";
 import { screeningQuestionsSectionHasMissingResponses } from "~/validators/profile";
+import { useUpdateApplicationMutation } from "~/api/generated";
+
+import { useNavigate } from "react-router-dom";
```

Comment:
Should be in import section above?

Reply:
Dang auto imports! 😆 

Fixed in [reorder imports](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6311/commits/820b060790d8c21a17c4e1bf46d49c112b0c832e)